### ISSUE TITLE
plugin-mainmenu: corresponding file choosers ignore single click settings of lxqt-config-input

### ISSUE THREAD

**Issue by pmattern** [lxde/lxqt#940]
File choosers "Choose {icon,menu} file" as invoked from plugin-mainmenu's configuration dialogue "Main Menu settings" are selecting folders and files upon single click no matter whether or not setting "Single click to activate items" in `lxqt-config-input` - Mouse is checked.

lxde/lxqt-panel@2c87224 on Arch Linux.

**Comment by palinek** [comment]
Confirmed.

I've digged a bit into this and it seems weird to me:
- the behaviour is/should be given by the QPA platform plugin, in our case `lxqt-qtplugin` -> here https://github.com/lxde/lxqt-qtplugin/blob/master/src/lxqtplatformtheme.cpp#L253; I've made a test if this hint is ever queried:
```diff
diff --git a/src/lxqtplatformtheme.cpp b/src/lxqtplatformtheme.cpp
index 679ed2a..e4a1715 100644
--- a/src/lxqtplatformtheme.cpp
+++ b/src/lxqtplatformtheme.cpp
@@ -251,6 +251,7 @@ QVariant LXQtPlatformTheme::themeHint(ThemeHint hint) const {
     case ToolBarIconSize:
         break;
     case ItemViewActivateItemOnSingleClick:
+qDebug() << "singleClickActivate_" << singleClickActivate_;
         return QVariant(singleClickActivate_);
     case SystemIconThemeName:
         return iconTheme_;
```
=> it is queried right upon the click on file name (in the dialog) => don't know why it doesn't work
- also checked the behaviour of `pcmanfm-qt`; in pcmafm-qt this particular hint isn't queried at all and this is because it is overrided by the application itself -> https://github.com/lxde/pcmanfm-qt/blob/master/pcmanfm/application.cpp#L60 (`pcmafm-qt` has its own configuration for the *singleClick*)

@PCMan may know more about this topic. Did you ever investigate the situation of the *singleClick* open?
